Package.swift

```diff
@@ -30,7 +30,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/IBM-Swift/LoggerAPI.git", from: "1.9.0"),
         .package(url: "https://github.com/apple/swift-log.git", Version("0.0.0") ..< Version("2.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "2.3.0"),
+        .package(url: "https://github.com/DanielMandea/Kitura-NIO.git", .branch("task/EPA-15899-make-buffer-public")),
```

Comment:
I approved https://github.com/DanielMandea/Kitura-NIO/pull/1 so we can merge it and change this reference to `master` (or maybe a tag?)